target/linux/x86/base-files.mk

```diff
@@ -0,0 +1,10 @@
+GRUB_SERIAL:=$(call qstrip,$(CONFIG_GRUB_SERIAL))
+ifeq ($(GRUB_SERIAL),)
+$(error This platform requires CONFIG_GRUB_SERIAL be set!)
+endif
+
+ifneq ($(GRUB_SERIAL),)
```

Comment:
Why do you check if `GRUB_SERIAL` is set here? It should error our before already.

Reply:
Dropped